### PR TITLE
Add ceph and its dependencies to Mariner

### PR DESCRIPTION
**Pull request by neha170**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [X] The toolchain has been rebuilt successfully (or no changes were made to it)
- [X] The toolchain/worker package manifests are up-to-date
- [X] Any updated packages successfully build (or no packages were changed)
- [X] All package sources are available
- [X] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [X] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [X] All source files have up-to-date hashes in the `*.signatures.json` files
- [X] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [X] Documentation has been updated to match any changes to the build system
- [X] Ready to merge

---

###### Summary 
Add ceph modules to mariner. Backported commit 4d16fcb from branch dev.

###### Change Log  
- Add ceph modules to mariner.
- No CVE's affected.

###### Does this affect the toolchain?  

YES

###### Test Methodology

- Local build

### CODE REVIEW

File: SPECS/CUnit/CUnit.spec [hunk 1]
```diff
@@ -0,0 +1,197 @@
+%global tarver 2.1-3
+
+Name:           CUnit
+Version:        2.1.3
+Release:        23%{?dist}
+Summary:        Unit testing framework for C
+Vendor:         Microsoft
```
Comment (PawelWMS) [inline SPECS/CUnit/CUnit.spec:7]: Please use "Microsoft Corporation" instead of "Microsoft".

File: SPECS/CUnit/CUnit.spec [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+%post -p /sbin/ldconfig
+%postun -p /sbin/ldconfig
+
+%files
+%{_datadir}/%{name}/
+%{_libdir}/libcunit.so.*
+%dir %{_docdir}/%{name}
+%{_docdir}/%{name}/AUTHORS
+%{_docdir}/%{name}/ChangeLog
+%{_docdir}/%{name}/COPYING
```
Comment (PawelWMS) [inline SPECS/CUnit/CUnit.spec:61]: Please use the `%license` macro for "COPYING". Same for other license-related files (if there are any more).

File: SPECS/Cython/Cython.spec [hunk 3]
```diff
@@ -1,79 +1,396 @@
...
+%{!?python3_sitelib: %define python3_sitelib %(python3 -c "from distutils.sysconfig import get_python_lib;print(get_python_lib())")}
+%define python3_sitearch %(python3 -c "from distutils.sysconfig import get_python_lib; import sys; sys.stdout.write(get_python_lib(1))")
+
+%global upname cython
+
+Name:           Cython
+Version:        0.29.13
+Release:        6%{?dist}
+Summary:        Language for writing Python extension modules
+Vendor:         Microsoft
```
Comment (PawelWMS) [inline SPECS/Cython/Cython.spec:10]: Same comment, please use "Microsoft Corporation". Please also update other spec files from this PR, if they have the same issue.

File: SPECS/Cython/Cython.spec [hunk 4]
```diff
@@ -1,79 +1,396 @@
...
+%setup -n %{upname}-%{version}
 
 %build
-python2 setup.py build
+python3 setup.py build
 
 %install
-python2 setup.py install --skip-build --root %{buildroot}
 
-%check
```
Comment (PawelWMS) [inline SPECS/Cython/Cython.spec:32]: Why are we removing the `%check` section? We shouldn't be removing tests, unless the new version of the package doesn't have them anymore.

File: SPECS/Cython/Cython.spec [hunk 5]
```diff
@@ -1,79 +1,396 @@
...
-*   Mon Apr 24 2017 Bo Gan <ganb@vmware.com> 0.25.2-1
--   Update to 0.25.2.
-*   Wed Oct 05 2016 ChangLee <changlee@vmware.com> 0.23.4-3
--   Modified %check.
-*   Tue May 24 2016 Priyesh Padmavilasom <ppadmavilasom@vmware.com> 0.23.4-2
--   GA - Bump release of all rpms.
-*   Wed Oct 28 2015 Divya Thaluru <dthaluru@vmware.com> 0.23.4-1
--   Initial build.
+* Fri Aug 21 2020 Thomas Crain <thcrain@microsoft.com> - 0.29.13-6
+- Initial CBL-Mariner import from Fedora 31 (License: MIT)
```
Comment (PawelWMS) [inline SPECS/Cython/Cython.spec:63]: Please also mention that this is replacing Photon's `Cython` package.

File: SPECS/Cython/Cython.spec [hunk 6]
```diff
@@ -1,79 +1,396 @@
...
-make %{?_smp_mflags} test
+python3 setup.py install --skip-build --root=%{buildroot}
+rm -rf %{buildroot}%{python3_sitelib}/setuptools/tests
 
-%clean
-rm -rf %{buildroot}
-
-%files
-%defattr(-,root,root,-)
+%files -n python3-%{name}
```
Comment (PawelWMS) [inline SPECS/Cython/Cython.spec:49]: It looks like this change is effectively removing the Photon's `Cython` package and adding a new `python3-Cython` package fom Fedora 31. While in core that shouldn't be a problem (I see you're already updating `python-lxml` to use the new package), I'm worried about derivatives, which stay with `Cython` as they will no longer receive any CVE fixes through no fault of their own unless they switch to the new package name. So I guess the question from my end is: do we know what is our policy for re-naming packages mid-release? Have you checked which (if any) derivatives depend on our current version of `Cython`? Fortunately, `Cython` doesn't seem to be a run-time requirement for anything and considering what this tool is for, I doubt derivatives make any use of it, but we must make sure.

File: SPECS/docbook5-schemas/docbook5-schemas.spec [hunk 7]
```diff
@@ -0,0 +1,397 @@
...
+Version:        5.1
+Release:        3%{?dist}
+License:        Freely redistributable without restriction
+URL:            https://docbook.org/
+Vendor:         Microsoft
+Distribution:   Mariner
+#Source0:       https://www.docbook.org/xml/%{version}/docbook-v%{version}-os.zip
+Source0:        %{name}-%{version}.zip
+Source1:        https://www.docbook.org/xml/5.0/docbook-5.0.zip
+Source2:        LICENSE
```
Comment (PawelWMS) [inline SPECS/docbook5-schemas/docbook5-schemas.spec:12]: >C [](start = 18, length = 1)

Where is this file coming from? If there is no licensing information available in the sources, but it's available elsewhere, we should mention where we got that information from. The `LICENSE.PTR` for the `gnu-efi` package is a good example and can be found under `SPECS/gnu-efi/LICENSE.PTR`. In short: in addition to the license info itself, please add a few lines inside the `LICENSE` file explaining why it's added separately and yet still represents the license information for this package.

**Review by PawelWMS — DISMISSED** [review]
:clock1:

File: SPECS/libnl3/libnl3.spec [hunk 8]
```diff
@@ -56,27 +63,35 @@ make %{?_smp_mflags} check
...
--	Updated build-requires after creating devel package for dbus.
-*	Tue Jun 23 2015 Divya Thaluru <dthaluru@vmware.com> 3.2.25-1
--	Initial build.
+* Thu Apr 30 2020 Nicolas Ontiveros <niontive@microsoft.com> - 3.4.0-4
+- Rename from libnl to libnl3.
+* Tue Apr 14 2020 Nick Samson <nisamson@microsoft.com> - 3.4.0-3
+- Updated Source0, URL. License verified.
+* Tue Sep 03 2019 Mateusz Malisz <mamalisz@microsoft.com> - 3.4.0-2
+- Initial import from Photon (license: dual Apache2/GPL2).
+* Wed Sep 19 2018 Bo Gan <ganb@vmware.com> - 3.4.0-1
```
Comment (PawelWMS) [inline SPECS/libnl3/libnl3.spec:86]: Please don't change this entry - the current `1.0-dev` version is correct.

File: SPECS/libstoragemgmt/libstoragemgmt.spec [hunk 9]
```diff
@@ -0,0 +1,911 @@
...
+mkdir -p %{buildroot}/%{_udevrulesdir}
+install -m 644 tools/udev/90-scsi-ua.rules \
+    %{buildroot}/%{_udevrulesdir}/90-scsi-ua.rules
+install -m 755 tools/udev/scan-scsi-target \
+    %{buildroot}/%{_udevrulesdir}/../scan-scsi-target
+
+mkdir -p %{buildroot}/%{_datadir}/bash-completion/completions/
+mv %{buildroot}/etc/bash_completion.d/lsmcli %{buildroot}/%{_datadir}/bash-completion/completions/
+
+%if 0%{with test}
```
Comment (PawelWMS) [inline SPECS/libstoragemgmt/libstoragemgmt.spec:228]: We have our own way of running the tests or not during build through the `RUN_CHECK` argument. We don't need the `%bcond_with     test` line and this `if` statement right here and they should be removed. Otherwise we will end up never running the tests, regardless of the `RUN_CHECK` argument's value.

File: SPECS/LICENSES-AND-NOTICES/LICENSE-LGPLv2.md [hunk 10]
```diff
@@ -0,0 +1,503 @@
+### GNU LESSER GENERAL PUBLIC LICENSE
```
Comment (PawelWMS) [inline SPECS/LICENSES-AND-NOTICES/LICENSE-LGPLv2.md:1]: Why do we need this file here?

File: SPECS/bison/bison.spec [hunk 11]
```diff
@@ -42,12 +46,16 @@ make %{?_smp_mflags} check
...
-* Sat May 09 00:21:04 PST 2020 Nick Samson <nisamson@microsoft.com> - 3.1-3
+* Fri Aug 21 2020 Thomas Crain <thcrain@microsoft.com> 3.1-4
+- Remove yacc command for compatibility with byacc package
+- Remove sha hash
+- License verified
+* Sat May 09 00:21:04 PST 2020 Nick Samson <nisamson@microsoft.com> 3.1-3
 - Added %%license line automatically
-
-*   Tue Sep 03 2019 Mateusz Malisz <mamalisz@microsoft.com> 3.1-2
--   Initial CBL-Mariner import from Photon (license: Apache2).
```
Comment (PawelWMS) [inline SPECS/bison/bison.spec:50]: Please revert to the change to this entry - it was correct and the new one is the outdated version.

Please also double-check this hasn't happened in other modified spec files.

File: SPECS/leveldb/leveldb.spec [hunk 12]
```diff
@@ -0,0 +1,208 @@
...
+# Add DB::SuspendCompactions() and DB:: ResumeCompactions() methods
+Patch0002:      0002-Added-a-DB-SuspendCompations-and-DB-ResumeCompaction.patch
+# Cherry-picked from Basho's fork
+Patch0003:      0003-allow-Get-calls-to-avoid-copies-into-std-string.patch
+# https://groups.google.com/d/topic/leveldb/SbVPvl4j4vU/discussion
+Patch0004:      0004-bloom_test-failure-on-big-endian-archs.patch
+
+BuildRequires:  cmake
+#Provided by gcc package
+#BuildRequires:  gcc-c++
```
Comment (PawelWMS) [inline SPECS/leveldb/leveldb.spec:26]: Please remove this and similar comments.

File: SPECS/python-docutils/python-docutils.spec [hunk 13]
```diff
@@ -51,21 +50,22 @@ python3 setup.py build
...
-mv %{buildroot}/%{_bindir}/rst2pseudoxml.py %{buildroot}/%{_bindir}/rst2pseudoxml3.py
-mv %{buildroot}/%{_bindir}/rst2odt_prepstyles.py %{buildroot}/%{_bindir}/rst2odt_prepstyles3.py
-mv %{buildroot}/%{_bindir}/rst2odt.py %{buildroot}/%{_bindir}/rst2odt3.py
-mv %{buildroot}/%{_bindir}/rst2man.py %{buildroot}/%{_bindir}/rst2man3.py
-mv %{buildroot}/%{_bindir}/rst2latex.py %{buildroot}/%{_bindir}/rst2latex3.py
-mv %{buildroot}/%{_bindir}/rst2html5.py %{buildroot}/%{_bindir}/rst2html53.py
-mv %{buildroot}/%{_bindir}/rst2html.py %{buildroot}/%{_bindir}/rst2html3.py
+for BINARY in rstpep2html rst2xml rst2xetex rst2s5 rst2pseudoxml rst2odt_prepstyles rst2odt rst2man rst2latex rst2html5 rst2html rst2html4
+do
+  mv %{buildroot}/%{_bindir}/$(echo $BINARY).py %{buildroot}%{_bindir}/$(echo $BINARY)3.py
```
Comment (PawelWMS) [inline SPECS/python-docutils/python-docutils.spec:58]: >$(echo $BINARY) [](start = 29, length = 15)

Why `$(echo $BINARY)` instead of only `$BINARY`? I'd suggest simplifying this here and elsewhere in this spec.

File: SPECS/python-docutils/python-docutils.spec [hunk 14]
```diff
@@ -107,18 +132,27 @@ popd
...
 %changelog
+* Fri Aug 21 2020 Thomas Crain <thcrain@microsoft.com> - 0.14-4
+- Follow the Fedora convention of providing a suffix-less binary name
+- License verified
+
 * Sat May 09 00:21:15 PST 2020 Nick Samson <nisamson@microsoft.com> - 0.14-3
 - Added %%license line automatically
 
-*   Tue Sep 03 2019 Mateusz Malisz <mamalisz@microsoft.com> 0.14-2
--   Initial CBL-Mariner import from Photon (license: Apache2).
```
Comment (PawelWMS) [inline SPECS/python-docutils/python-docutils.spec:116]: Another changelog entry to revert.

File: SPECS/python-lxml/python-lxml.spec [hunk 15]
```diff
@@ -40,46 +35,34 @@ Python 3 version.
...
 %defattr(-,root,root,-)
+%license LICENSES.txt
 %{python3_sitelib}/*
 
 %changelog
-* Sat May 09 00:20:50 PST 2020 Nick Samson <nisamson@microsoft.com> - 4.2.4-6
-- Added %%license line automatically
-
+*   Wed Aug 26 2020 Thomas Crain <thcrain@microsoft.com> 4.2.4-7
+-   Remove python2 support.
```
Comment (PawelWMS) [inline SPECS/python-lxml/python-lxml.spec:63]: Please also check the license and add the `License verified.` entry to the changelog, since we're adding the `%license` macro.

Btw. thank you for doing that (either to you or Thomas). :)

File: SPECS/libstoragemgmt/libstoragemgmt.spec [hunk 16]
```diff
@@ -0,0 +1,911 @@
+%{!?python3_sitelib: %define python3_sitelib %(python3 -c "from distutils.sysconfig import get_python_lib;print(get_python_lib())")}
+
+%bcond_with     test
```
Comment (hbeberman) [inline SPECS/libstoragemgmt/libstoragemgmt.spec:3]: Remove this as well when you remove the %if test around the %check section

File: SPECS/CUnit/CUnit.spec (hunk 1 above)
Comment (neha170) [reply SPECS/CUnit/CUnit.spec:7]: Done

File: SPECS/CUnit/CUnit.spec (hunk 2 above)
Comment (neha170) [reply SPECS/CUnit/CUnit.spec:61]: Done

File: SPECS/Cython/Cython.spec (hunk 3 above)
Comment (neha170) [reply SPECS/Cython/Cython.spec:10]: Done

File: SPECS/Cython/Cython.spec (hunk 5 above)
Comment (neha170) [reply SPECS/Cython/Cython.spec:63]: The spec file has the following code:
`Provides:       %{name} = %{version}-%{release}  `
`Provides:       %{name}%{?_isa} = %{version}-%{release}  `
`Obsoletes:      %{name} < %{version}-%{release}`

File: SPECS/libnl3/libnl3.spec (hunk 8 above)
Comment (neha170) [reply SPECS/libnl3/libnl3.spec:86]: Done

File: SPECS/bison/bison.spec (hunk 11 above)
Comment (neha170) [reply SPECS/bison/bison.spec:50]: Done, reverted the change in python-docutils.spec also

File: SPECS/leveldb/leveldb.spec (hunk 12 above)
Comment (neha170) [reply SPECS/leveldb/leveldb.spec:26]: Done

File: SPECS/LICENSES-AND-NOTICES/LICENSE-LGPLv2.md (hunk 10 above)
Comment (neha170) [reply SPECS/LICENSES-AND-NOTICES/LICENSE-LGPLv2.md:1]: The spec file for ceph was taken from Ceph source, under the LGPLv2 license.
`Initial CBL-Mariner import from Ceph source (license: LGPLv2.1)`
I have added the license in the license.json file:
`{`
           ` "origin": "Ceph",`
         `   "license": "[LGPL2.1](https://github.com/ceph/ceph/blob/master/COPYING-LGPL2.1)",`
           ` "specs": [`
              `  "ceph"`
        `    ]`
      `  }`

File: SPECS/python-docutils/python-docutils.spec (hunk 13 above)
Comment (neha170) [reply SPECS/python-docutils/python-docutils.spec:58]: I checked with just putting '%{_bindir}/$BINARY.py'. It gets interpreted as 'usr/bin/.py'. So, I am leaving it as '%{_bindir}/$(echo $BINARY).py'

File: SPECS/python-lxml/python-lxml.spec (hunk 15 above)
Comment (neha170) [reply SPECS/python-lxml/python-lxml.spec:63]: Done

File: SPECS/libstoragemgmt/libstoragemgmt.spec (hunk 16 above)
Comment (neha170) [reply SPECS/libstoragemgmt/libstoragemgmt.spec:3]: Done.

File: SPECS/libstoragemgmt/libstoragemgmt.spec (hunk 9 above)
Comment (neha170) [reply SPECS/libstoragemgmt/libstoragemgmt.spec:228]: Done.

File: SPECS/byacc/byacc.spec [hunk 17]
```diff
@@ -0,0 +1,305 @@
+%define byaccdate 20200330
+
+Summary:      Berkeley Yacc, a parser generator
+Name:         byacc
+Version:      1.9.%{byaccdate}
+Release:      3%{?dist}
+License:      Public Domain
+URL:          http://invisible-island.net/byacc/byacc.html
```
Comment (thcrain-msft) [inline SPECS/byacc/byacc.spec:8]: >http://invisible-island.net/byacc/byacc.html [](start = 14, length = 44)

Nit: This could be https

File: SPECS/leveldb/leveldb.spec [hunk 18]
```diff
@@ -0,0 +1,208 @@
...
+%make_build
+
+%install
+%make_install
+
+mkdir -p %{buildroot}%{_libdir}/pkgconfig
+cp -a %{name}.pc %{buildroot}%{_libdir}/pkgconfig/
+
+%check
+%ctest
```
Comment (thcrain-msft) [inline SPECS/leveldb/leveldb.spec:72]: Do we actually support the `ctest` macro?

File: SPECS/python-cherrypy/python-cherrypy.spec [hunk 19]
```diff
@@ -0,0 +1,67 @@
...
+
+%files -n python3-%{pkgname}
+%license LICENSE.md
+%doc README.rst docs/
+%{python3_sitelib}/*
+%{_bindir}/cherryd
+
+%changelog
+* Fri Aug 21 2020 Thomas Crain <thcrain@microsoft.com> 1.4.0-1
+- Original CBL-Mariner version
```
Comment (thcrain-msft) [inline SPECS/python-cherrypy/python-cherrypy.spec:66]: Nit: Can we change this (and any similar lines in this PR) to "Original version for CBL-Mariner"? It seems to be the standard wording nowadays

File: SPECS/yasm/yasm.spec [hunk 20]
```diff
@@ -0,0 +1,230 @@
+Summary:        Modular Assembler
+Name:           yasm
+Version:        1.3.0
+Release:        13%{?dist}
+License:        BSD and (GPLv2+ or Artistic or LGPLv2+) and LGPLv2
+URL:            https://yasm.tortall.net/
+Vendor:         Microsoft
+Distribution:   Mariner
+Source0:        http://www.tortall.net/projects/%{name}/releases/%{name}-%{version}.tar.gz
```
Comment (thcrain-msft) [inline SPECS/yasm/yasm.spec:9]: >http://www.tortall.net/projects/%{name}/releases/%{name}-%{version}.tar.gz [](start = 16, length = 74)

Please switch to the HTTPS version, since it works on this site

File: SPECS/xmlstarlet/xmlstarlet.spec [hunk 21]
```diff
@@ -0,0 +1,208 @@
...
+autoreconf -i
+%configure --disable-static-libs --with-libxml-include-prefix=%{_includedir}/libxml2 --docdir=%{_pkgdocdir} # --libdir=%{_libdir}
+make %{?_smp_mflags}
+
+
+%install
+rm -fr %{buildroot}
+make install DESTDIR=$RPM_BUILD_ROOT
+# Avoid name kludging in autotools
+mv $RPM_BUILD_ROOT%{_bindir}/xml $RPM_BUILD_ROOT%{_bindir}/xmlstarlet
```
Comment (thcrain-msft) [inline SPECS/xmlstarlet/xmlstarlet.spec:48]: >$RPM_BUILD_ROOT [](start = 3, length = 15)

Please change instances of `$RPM_BUILD_ROOT` to `${buildroot}`, since they're equivalent and we've standardized on the shorter version.

File: SPECS/python-yamlloader/python-yamlloader.spec [hunk 22]
```diff
@@ -0,0 +1,90 @@
...
+Distribution:   Mariner
+#Source0:       https://github.com/Phynix/%{pkgname}/archive/%{version}.tar.gz
+Source0:        %{pkgname}-%{version}.tar.gz
+BuildArch:      noarch
+
+%description 
+This module provides loaders and dumpers for PyYAML. 
+
+
+%if %{with python2}
```
Comment (thcrain-msft) [inline SPECS/python-yamlloader/python-yamlloader.spec:23]: I didn't realize at the time, but this is a Fedora relic that we don't use and don't plan to ever use in 1.0. Any new python packages in this PR should have been made without Python2 support (and if they were, they can be safely removed). Can we remove any instances of these python2 versions (for new packages only) in this PR?

**Review by thcrain-msft — CHANGES_REQUESTED** [review]
:clock1:

File: SPECS/byacc/byacc.spec (hunk 17 above)
Comment (neha170) [reply SPECS/byacc/byacc.spec:8]: Done

File: SPECS/python-cherrypy/python-cherrypy.spec (hunk 19 above)
Comment (neha170) [reply SPECS/python-cherrypy/python-cherrypy.spec:66]: Done, changed for all the files currently in SPECS folder

File: SPECS/yasm/yasm.spec (hunk 20 above)
Comment (neha170) [reply SPECS/yasm/yasm.spec:9]: Done

File: SPECS/xmlstarlet/xmlstarlet.spec (hunk 21 above)
Comment (neha170) [reply SPECS/xmlstarlet/xmlstarlet.spec:48]: Done, changed it in all the files currently in the PR

File: SPECS/leveldb/leveldb.spec (hunk 18 above)
Comment (neha170) [reply SPECS/leveldb/leveldb.spec:72]: Removed the macro. Replaced it with 'ctest -V %{?_smp_mflags}'

File: SPECS/python-yamlloader/python-yamlloader.spec (hunk 22 above)
Comment (neha170) [reply SPECS/python-yamlloader/python-yamlloader.spec:23]: Removed all instances in this PR.

File: SPECS/docbook5-schemas/docbook5-schemas.spec (hunk 7 above)
Comment (neha170) [reply SPECS/docbook5-schemas/docbook5-schemas.spec:12]: Removed the file, added a '%license docbook-v5.1-os.html' in the files section.

File: cgmanifest.json [hunk 23]
```diff
@@ -840,13 +860,23 @@
...
+    },
     {
       "component": {
         "type": "other",
         "other": {
           "name": "Cython",
-          "version": "0.28.5",
-          "downloadUrl": "https://github.com/cython/cython/archive/0.28.5.tar.gz"
+          "version": "0.29.13",
+          "downloadUrl": "https://github.com/cython/cython/archive/0.29.13.tar.gz"
```
Comment (hbeberman) [inline cgmanifest.json:879]: Revert this change to cgmanifest since this PR doesn't update Cython any more.

File: SPECS/LICENSES-AND-NOTICES/data/licenses.json [hunk 24]
```diff
@@ -17,13 +17,17 @@
...
                 "busybox",
+                "byacc",
                 "calamares",
                 "catch",
                 "chrony",
                 "conntrack-tools",
                 "cpprest",
                 "cryptsetup",
+                "CUnit",
+                "Cython",
```
Comment (hbeberman) [inline SPECS/LICENSES-AND-NOTICES/data/licenses.json:28]: Don't add cython here since you're not modifying it.

File: SPECS/LICENSES-AND-NOTICES/data/licenses.json [hunk 25]
```diff
@@ -296,7 +325,6 @@
                 "ctags",
                 "curl",
                 "cyrus-sasl",
-                "Cython",
```
Comment (hbeberman) [inline SPECS/LICENSES-AND-NOTICES/data/licenses.json:299]: Don't remove cython here since you're not modifying it.

File: .github/workflows/validate-cg-manifest.sh [hunk 26]
```diff
@@ -13,6 +13,8 @@
 
 # Ignore some specs, mostly those with Source0 files that are not from an external source, or have very odd URLs
 ignore_list=" \
+  Cython \
```
Comment (hbeberman) [inline .github/workflows/validate-cg-manifest.sh:16]: You should remove this since you arent updating Cython

File: cgmanifest.json (hunk 23 above)
Comment (neha170) [reply cgmanifest.json:879]: Done

File: SPECS/LICENSES-AND-NOTICES/data/licenses.json (hunk 24 above)
Comment (neha170) [reply SPECS/LICENSES-AND-NOTICES/data/licenses.json:28]: Done

File: SPECS/LICENSES-AND-NOTICES/data/licenses.json (hunk 25 above)
Comment (neha170) [reply SPECS/LICENSES-AND-NOTICES/data/licenses.json:299]: Done

File: .github/workflows/validate-cg-manifest.sh (hunk 26 above)
Comment (neha170) [reply .github/workflows/validate-cg-manifest.sh:16]: Done

**Review by hbeberman — APPROVED** [review]

File: .github/workflows/validate-cg-manifest.sh [hunk 27]
```diff
@@ -23,6 +24,12 @@ ignore_list=" \
   python-markupsafe \
   python-requests \
   python-zope-interface \
+  openjdk8 \
```
Comment (thcrain-msft) [inline .github/workflows/validate-cg-manifest.sh:27]: Looks like the auto merge added this- we take our openjdk8 sources from a regular place now, so we can remove it from this list I believe

**Review by thcrain-msft — APPROVED** [review]
Looks good to me! Just one note.

File: .github/workflows/validate-cg-manifest.sh (hunk 27 above)
Comment (neha170) [reply .github/workflows/validate-cg-manifest.sh:27]: Done
